src/cohort_manager.js

```diff
@@ -0,0 +1,117 @@
+const Cohorts = require('./cohorts.js')
+
+class CohortManager {
+  constructor() {
+    this.cohortList = []
+    this.studentId = 1
+    this.max_capacity = 5
+  }
+
+  createCohort(cohortName) {
+    const cohortToAdd = new Cohorts(cohortName)
+    this.cohortList.push(cohortToAdd)
+    return this.cohortList
+  }
+
+  searchByCohortName(cohortName) {
+    let cohortFound = false
+    let cohort
+    for (let i = 0; i < this.cohortList.length; i++) {
+      if (this.cohortList[i].name === cohortName) {
+        cohortFound = true
+        cohort = this.cohortList[i]
+        break
+      }
+    }
+    if (cohortFound === false) {
+      return 'COHORT NOT FOUND'
```

Comment:
Instead of writing out the errors one by one you could do something like this at the start of the file: 
`const cohortNotFoundError = 'Cohort not found.'
const cohortFullError = 'This cohort is full.'
const cohortCannotExistWithoutNameError =
  'Please name the cohort before initialising.'
const cohortAlreadyExistsError = 'This cohort already exists.'
const studentNotFoundError = 'Student not found.'
const studentInAnotherCohortError = 'This student is already in another cohort.'`

And then instead of `return 'COHORT NOT FOUND'`, you could do:
`return cohortNotFoundError`

May make things easier in the long run, I found it helpful